import/import.go

```diff
@@ -0,0 +1,534 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"os/exec"
+	"strings"
+
+	"golang.org/x/exp/maps"
+
+	"github.com/astronomer/terraform-provider-astro/internal/clients/iam"
+
+	"github.com/astronomer/terraform-provider-astro/internal/clients"
+
+	"github.com/astronomer/terraform-provider-astro/internal/clients/platform"
+	"github.com/samber/lo"
+)
+
+type HandlerResult struct {
+	Resource string
+	Error    error
+}
+
+func main() {
+	log.SetFlags(0)
+	log.Println("Terraform Import Script Starting")
+
+	// collect all arguments from the user, indicating all the resources that need to be imported
+	resourcesPtr := flag.String("resources", "", "Comma separated list of resources to import")
+	tokenPtr := flag.String("token", "", "API token to authenticate with the platform")
+	hostPtr := flag.String("host", "https://api.astronomer.io", "API host to connect to")
+	organizationIdPtr := flag.String("organizationId", "", "Organization ID to import resources into")
+
+	flag.Parse()
+
+	*resourcesPtr = strings.ToLower(*resourcesPtr)
+	resources := strings.Split(*resourcesPtr, ",")
+	log.Println("Resources to import: ", resources)
+
+	// set the API token
+	token := *tokenPtr
+	if token == "" {
+		token = os.Getenv("ASTRO_API_TOKEN")
+	}
+
+	if len(token) == 0 {
+		log.Fatal("API token not provided")
+		return
+	}
+
+	// set the host
+	var host string
+	if *hostPtr == "dev" {
+		host = "https://api.astronomer-dev.io"
+	} else if *hostPtr == "stage" {
+		host = "https://api.astronomer-stage.io"
+	} else {
+		host = *hostPtr
+	}
+
+	// set the organization ID
+	organizationId := *organizationIdPtr
+	if organizationId == "" {
+		log.Fatalf("Organization ID not provided")
+	}
+
```

Comment:
we should make sure they have terraform installed early by checking `terraform -version` here and erroring with a good err message if now